instance/permissions/models.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app permission models
+"""
+
+# Imports #####################################################################
+
+from django.db import models
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+
+
+# Models ######################################################################
+
+class InstanceManagerPermissionManager(models.Manager):
+    """
+    Model manager for InstanceManagerPermission.
+    """
+    content_type_model = 'instance_manager_permission'
+
+    def get_queryset(self):
+        """Returns the InstanceManagerPermission instance for this content type model."""
+        return (super(InstanceManagerPermissionManager, self)  # pylint: disable=no-member
+                .get_queryset()
+                .filter(content_type__model=self.content_type_model,))
+
+
+class InstanceManagerPermission(Permission):
+    """
+    Elevated privilege permission which spans all instance models.
+    """
+    objects = InstanceManagerPermissionManager()
+
+    class Meta:
+        proxy = True
+        verbose_name = InstanceManagerPermissionManager.content_type_model
+
+    def save(self, *args, **kwargs):
+        """Use a proxy content type called instance.instance_manager_permission"""
+        ct, dummy = ContentType.objects.get_or_create(  # pylint: disable=no-member
+            app_label=self._meta.app_label, model=self._meta.verbose_name,
+        )
```

Comment:
When I wrote the task description, I didn't realise that creating a global permission in Django is *more* complicated than creating a permission attached to a model.  I tried to keep it as simple as possible, but I guess it isn't too bad anyway. :)